### PR TITLE
Update scheduler arguments and copy experiment info

### LINKED ISSUE #240: Show arguments in scheduler neatly
### Feature you want to implement
Now, the scheduler app shows all the decimal points of arguments, it may be too long.
Thus, I will show them neatly.

In addition, I will make an user copy the specific submitted experiment info.

### How the feature is implemented
To allow copy, I will use [QClipboard](https://doc.qt.io/qtforpython-5/PySide2/QtGui/QClipboard.html).

### PR DESCRIPTION
**Pull request by BECATRUE**
I implemented the followings:
- Show number arguments up to 9 decimal points (ns scale)
- Make an user copy a specific experiment info through right-click

The example result is as below.

image

This closes #240.

### CODE REVIEW

File: iquip/apps/scheduler.py [hunk 1]
```diff
@@ -256,9 +276,27 @@ def __init__(self, name: str, parent: Optional[QObject] = None):
...
+    def setCopyAction(self):
+        """Sets a copy action in schedulerFrame.scheduleView."""
+        view = self.schedulerFrame.scheduleView
+        action = QAction("Copy", view)
+        action.triggered.connect(self.copyExperimentInfo)
+        view.addAction(action)
+
+    @pyqtSlot()
+    def copyExperimentInfo(self):
+        """Copies the selected experiment info to the system clipboard."""
```
Comment (kangz12345) [inline iquip/apps/scheduler.py]: Is it possible to copy the item at the selected column only?
For example only the RID or the arguments, etc.

Comment (BECATRUE) [reply iquip/apps/scheduler.py]: Sure! Indeed, actions can be applied to each index i.e., row and column, respectively.
However, I thought we don't need to copy each item, hence I applied same actions to all columns.

According to your suggestion, I will make one copy each item.
And I will add a function that copy the specific experiment info when right-click a row number.

Comment (kangz12345) [reply iquip/apps/scheduler.py]: Is the first column with indices the header?

Comment (BECATRUE) [reply iquip/apps/scheduler.py]: ~~No! The columns indicate only the table contents, and we can obtain the header through `view.verticalHeader()`.
Please refer to [setCopyAction](https://github.com/snu-quiqcl/iquip/blob/BECATRUE/240/update-scheduler-arguments/iquip/apps/scheduler.py#L287).~~

Yes! It means the header.

**Review by kangz12345 — APPROVED** [review]

**Pull request merged**